scripts/protocgen.sh

```diff
@@ -36,5 +36,5 @@ go mod tidy
 
 
 # move proto files to the right places
-cp -r github.com/cosmos/ibc-go/v*/modules/* modules/
+cp -r github.com/line/ibc-go/v3/modules/* modules/
```

Comment:
```suggestion
cp -r github.com/line/ibc-go/v*/modules/* modules/
```
How abount `v*`?

Reply:
What's happening, when there are same files in v*(v1, v2, v3)?
Do they override with the last(files in v3)? It seems to override with the last.

In the case that we have a file A in v2 folder and a file A get removed since v3. 
In this context, Is it okay to copy file A to modules directory?

Reply:
> In the case that we have a file A in v2 folder and a file A get removed since v3.
> In this context, Is it okay to copy file A to modules directory?

Certainly this could lead to confusion. It looks good with v3.

Reply:
Thanks for the good suggestion. The following pr already has that applied. Thank you.